packages/blockchain-link/src/workers/solana/tokenUtils.ts

```diff
@@ -1,4 +1,19 @@
 import { TokenDetailByMint } from '@trezor/blockchain-link-types/lib/solana';
 
-export const getTokenMetadata = async (): Promise<TokenDetailByMint> => Promise.resolve({});
+const TOKEN_WHITELIST = {
```

Comment:
These are just some common tokens we've used in testing, this is meant as a baseline to be extended if we decide for the whitelist approach